go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -220,22 +220,50 @@ func (npw *nodePortWatcher) updateServiceFlowCache(service *kapi.Service, add, h
 		// Flows for cloud load balancers on Azure/GCP
```

Comment:
I was reading the whole function and I saw that the table numbers between comments and code are not always aligned. I suppose this is ok if it hasn't been fixed so far, right?
For instance `// table 0, This rule matches on all traffic with dst port == NodePort, DNAT's the nodePort to the svc targetPort`  but then we modify table 6. Is this a problem at all?